config/default.schema.json

```diff
@@ -27,10 +27,15 @@
 				"TIMEOUT_MS": {
 					"type": "integer",
 					"description": "Outgoing request timeout in milliseconds."
+				},
+				"TIMEOUT_ES": {
```

Comment:
We need the "TIMEOUT_ES" for requesting the H5P content with the help of the Edu-Sharing Rendering Service. Several H5P contents are large files, so that the request needs up to 8 seconds before the content is shown.
Should I rename the TIMEOUT to "TIMEOUT_ES_MS"?

Reply:
"Edu-Sharing requests take a lot of time" ..sound not good. Do we know the reasons? Let speak with @Loki-Afro 
Because the TIMEOUT_MS is high for request. But if it need longer then the code can not be moved to nest. Otherwise our interceptor will kill this request. If we by pass it our network will kill the request latlly after 30 seconds in legacy and nestjs stack.

Reply:
in addition to that, if there is really a timeout needed .. always when adding a variable with time, use the damn timeunit